security/subsystem/src/main/java/org/jboss/as/security/remoting/RemotingLoginModule.java

```diff
@@ -123,7 +123,7 @@ public boolean login() throws LoginException {
                         SSLSession session = con.getSslSession();
                         if (session != null) {
                             try {
-                                credential = session.getPeerCertificateChain()[0];
+                                credential = session.getPeerCertificates()[0];
```

Comment:
@darranl Do we really need a new configuration option in upstream? Is it a new feature or a bug in the previous implementation? Either way, it cannot go to a 6.4 CP as we cannot change API there and cannot upgrade schema as well.
If we decide to introduce a new configuration option, I will implement it.